help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,94 @@ title: "Configure authentication"
+You can enable all authentication methods separately or combined.
 
-[Learn more about authentification](../govern/authentication.md).
+If supported authentication methods do not work for you, contact us on <info@datagrok.ai> or [our community](https://community.datagrok.ai/) and we will discuss options for your specific case.
 
-## Common options
+[Learn more about authentification](../../govern/authentication.md).
 
-If you want to display something on the login window, set the 'Prompt' option
-![Login prompt](login-prompt.png)
+## Common options
 
-Enable the 'Send Welcome Email' option to send welcome emails to new signed-up users.
+You can display a text on the login window, like greetings words etc.
+To do so set the *Prompt* option in **Settings** > **Users and Sessions** > **General**:
+<p align="center">
+  <img alt="login-prompt" src={require('./login-prompt.png').default} width="300px"/>
+</p>
 
 ## Login-password authentication
 
-It is the most basic method to authenticate users with Datagrok.
-
 To configure login-password authentication for Datagrok:
 
-1. Go to the Datagrok Settings section 'Users and Sessions'; this section contains all authentication settings.
-2. Enable 'Internal authentication' to use the login-password method
-3. Signup option is alterable; you can disable it using the 'Signup Allowed' option
-4. To restrict from which domains people can sign up to the platform, use the 'Signup Domains Whitelist' option. You can
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears.
+2. Enable *Internal authentication* to use the login-password method.
+3. Adjust the *Signup* option. You can disable or enable it using the *Signup Allowed* and *Allow Oauth Signup* checkboxes.
+4. For better security you can restrict from which domains people can sign up to the platform. To do so use the *Signup Domains Whitelist* option. You can
    set several domains separated with commas
-5. Enable the 'Require Email Confirm' option to force people to use working emails.
+5. Enable the *Send Welcome Email* option to send welcome emails to new signed-up users.
+6. Enable the *Require Email Confirm* option to force users to use working emails.
+
+To use login-password authentication, it is important to configure an [email service](configure-smtp.md) that delivers signup and password emails.
 
-### Email service
+## OAuth authentication
 
-For login-password authentication, it is important to configure an Email service that will deliver signup, welcome,
-confirmation and forgot password emails.
+Datagrok supports [Google](#google-authentication), [Facebook](#facebook-authentication) and [GitHub](#github-authentication) OAuth authentication.
 
-[Learn how to configure SMTP for Datagrok](configure-smtp.md).
+### Google authentication
 
-## Oauth authentication
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears
```

Comment:
I think we can mention it one on the top of the page, instead of writing it every time